builder-image-gitops/go/src/gitops/main.go

```diff
@@ -73,8 +73,15 @@ func gitCommit(worktree *git.Worktree, file string) (err error) {
 		return
 	}
 
+  val,exists:= os.LookupEnv("GIT_URL")
+  if exists {
+    source := strings.Split(strings.SplitN(val, "/", 4)[3], ".")
+    val = " from " + source[0]
+  }
+
```

Comment:
Can you refactor this so that GIT_URL is only parsed once in the main function and passed into sub functions